RELEASE.md

```diff
@@ -11,6 +11,8 @@
 * Changed the execution of `SequentialRunner` to not use an executor pool to ensure it's single threaded.
 
 ## Breaking changes to the API
+* Private methods `_is_project`, `_find_kedro_project`, `_split_params` are made public.
```

Comment:
This section corresponds to the Kedro 0.19.12 (non breaking release), I would add a section on top for Kedro 1.0.0 separately and add this change there